Core/StitchCoreSDK/Sources/StitchCoreSDK/Internal/Net/RawSSEStream.swift

```diff
@@ -0,0 +1,148 @@
+import Foundation
+
+private let newlineChar = [UInt8]("\n".utf8)[0]
+
+public enum SSEStreamState {
+    case open, closed
+}
+
+open class SSEStreamDelegate {
```

Comment:
[style q] why is this an open class instead of a protocol?

Reply:
Seemed like a more relevant pattern here. Not every method is going to need to be overridden. Apple uses this pattern behind the scenes a decent amount. There may have been another reason that I do not recall.